.github/workflows/main.yml

```
@@ -21,9 +26,22 @@ jobs:
         DESTDIR: out
       run: |
         meson build
-        ninja -C build
         ninja -C build install
-              
+
+    - name: Build (Fedora)
+      env:
+        DESTDIR: out
+      run: |
+        meson configure -Dcurated=false -Dhomepage=false -Dpayments=false  build
```

Since the `homepage` build flag was dropped:

```suggestion
        meson configure -Dcurated=false -Dpayments=false build
```